src/appkit.cs

```diff
@@ -23545,7 +23540,6 @@ interface NSFilePromiseProvider : NSPasteboardWriting
 
 	[Mac (10,12)]
 	[Protocol]
-	[BaseType (typeof(NSObject))]
```

Comment:
Right now we're providing `[Model]` for _most_ `*Delegate` type - but not others (because they are not often useful as concrete types and used as interfaces). 

If we had to start from scratch I would not want any model ;-) as our IDE can generate them when needed.